src/DependencyInjection/CsaGuzzleExtension.php

```diff
@@ -95,18 +96,12 @@ private function processCacheConfiguration(array $config, ContainerBuilder $cont
     private function processClientsConfiguration(array $config, ContainerBuilder $container, Definition $clientFactory, Definition $descriptionFactory)
     {
         foreach ($config['clients'] as $name => $options) {
-            $clientFactory->addMethodCall('registerClientConfiguration', [
-                $name,
-                $options['config'],
-                $options['subscribers']
+            $client = new Definition($config['factory_class']);
+            $client->addArgument($options['config']);
+            $client->addTag(SubscriberPass::CLIENT_TAG, [
+                'subscribers' => implode(', ', $options['subscribers'])
```

Comment:
Any news on this, @kgilden? Don't hesitate to tell, if you run into any trouble! :)